python/python/glide/async_commands/core.py

```diff
@@ -294,6 +294,25 @@ async def strlen(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.Strlen, [key]))
 
+    async def rename(self, key: str, new_key: str) -> TOK:
+        """
+        Renames `key` to `new_key`.
+        If `newkey` already exists it is overwritten.
+        In Cluster mode, both `key` and `newkey` must be in the same hash slot,
```

Comment:
Consider to split implementation for cluster and standalone clients with different docs